src/govee_ble/parser.py

```diff
@@ -87,13 +87,13 @@ def _process_mfr_data(
         service_uuids: list[str],
     ) -> None:
         """Parser for Govee sensors."""
-        _LOGGER.debug("Parsing Govee sensor: %s %s", mgr_id, data)
+        _LOGGER.debug("Parsing Govee sensor: %s %s", mgr_id, self.hex(data))
```

Comment:
The new function will get run and thrown away even if debug logging is disabled because python has to build the result and send it to the debug call which will than reject it with the isEnabledFor check

you can avoid this by checking isEnabledFor before calling debug